### PR TITLE
Topic/info

### LINKED ISSUE #40: Update SMA_INFO output
The SMA_INFO output in src/init.c is missing some environment variables.  For example, CMA variables should be output when CMA is included in the build.

### PR DESCRIPTION
**Pull request by jdinan**
Closes #40 and #176

### CODE REVIEW

**Comment by jdinan** [comment]
@jpdoyle -- your turn to review.  :)

File: src/transport_ofi.c [hunk 1]
```diff
@@ -1176,6 +1176,32 @@ int shmem_transport_startup(void)
...
+    printf("SMA_OFI_FABRIC          %s\n",
+           (shmem_util_getenv_str("OFI_FABRIC") != NULL) ?
+           shmem_util_getenv_str("OFI_FABRIC") : "AUTO");
+    printf("\tFabric that should be used by the OFI transport\n");
+    printf("SMA_OFI_DOMAIN          %s\n",
+           (shmem_util_getenv_str("OFI_DOMAIN") != NULL) ?
+           shmem_util_getenv_str("OFI_DOMAIN") : "AUTO");
+    printf("\tFabric domain that should be used by the OFI transport\n");
+    printf("SMA_OFI_ATOMIC_CHECKS_WARN %s\n",
+           (NULL != shmem_util_getenv_str("OFI_ATOMIC_CHECKS_WARN")) ?
```
Comment (jpdoyle) [inline src/transport_ofi.c]: You use both (NULL != x) and (x != NULL) here. We should stick to a consistent choice, at least within each file. I prefer (x != NULL) for readability, but (NULL != x) is less dangerous if typo'd, and (NULL != x) is already used elsewhere in transport_ofi.c

File: src/transport_portals4.c [hunk 2]
```diff
@@ -649,6 +649,15 @@ shmem_transport_startup(void)
 }
 
 
+void shmem_transport_print_info(void)
+{
+    char *ofi_provider;
+
+    printf("\n");
+    printf("Network transport:      OFI\n");
```
Comment (jpdoyle) [inline src/transport_portals4.c]: This is a lie.

File: src/transport_ofi.c [hunk 3]
```diff
@@ -1176,6 +1176,32 @@ int shmem_transport_startup(void)
...
+           "Set" : "Not set");
+    printf("\tDisplay warnings about unsupported atomic operatios\n");
```
Comment (jpdoyle) [inline src/transport_ofi.c]: Typo. s/operatios/operations/

**Comment by jdinan** [comment]
Thanks!  I pushed a patch to fix up the things you found, and will merge once Travis gives the thumbs-up.

**Pull request merged**